### DISCUSSION TITLE
Compare send request block

### DISCUSSION THREAD

**Discussion by zesagata** [postmanlabs/postman-flows#129] [Q&A] [answered]
Can we compare the output of 2 send request block? 

Something like this
![image](https://user-images.githubusercontent.com/24735004/142606191-72c2ea5d-29f5-454a-8298-90eadebfbca2.png)

**Comment by samay-v** [comment, marked as answer]
Hey @zesagata 
It isn't possible with the validate block right now. The validate block compares an input with a user entered constant only. 
Your use-case should be possible with the [Expression input and the check block](https://github.com/postmanlabs/postman-flows/discussions/124) coming in

**Comment by zesagata** [comment]
I see, thanks for the response!
